autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/generateSchedulesMock.ts

```diff
@@ -7,6 +8,8 @@ interface MockResponse {
 export default async function fetch(path: string, init?: RequestInit): Promise<MockResponse> {
   if (path === '/api/scheduling/generate' && init) {
     const schedules = [testSchedule1, testSchedule2, testSchedule3];
+    // DEBUG
+    await wait(2000);
```

Comment:
Remind me to get rid of this before merging

Reply:
Typically for stuff like this, I make a commit that starts with `(temp)` only containing the lines I want to drop at the end of it, then with Git Graph you can right click the commit and drop it (or interactive rebase but I think this is easier). Also helps me remember that there's something to drop just given the `(temp)` message